scripts/manual_reviewer_smoke_test.py

```
@@ -0,0 +1,325 @@
+        f"--port",
+        str(PORT),
+        f"--conf",
+        "mode",
+        "review",
+    ]
+
+    print(f"Command: {' '.join(cmd)}")
+    print()
+
+    # Start server in background
+    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True, cwd=str(repo_root))
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)